CHECK_POWER_COMMANDS/Check_Power_Cmds.py

```diff
@@ -201,32 +205,40 @@ def run_one_command(cmd, system_state, bfc, rule_set):
 # Grab the first item in the array
 present_cmd = system_packets[array_row_number]
 
-# Now  process each entry until you hit your
-# first ACISPKT.  Then you process that first one.  This will set up
-# the system state for processing the next ACISPKT that you encounter
-while present_cmd['cmd_type'] != 'ACISPKT':
 
-    # Run the ORB rules on the present state
-    system_state, new_rules_fired, violations_list = run_one_command(present_cmd,
-                                                                     system_state,
-                                                                     bfc,
-                                                                     Rulesets.ORB_CMD_SW_rule_set)
+# If this s a Vehicle-Only Review load then there will be no ACISPKT commands within the load.
+# So skip the while loop which is intended ot take you to the first one.  
+# This means that array_row_number will equal 0
+
+if 'ACISPKT' in system_packets['cmd_type']:
```

Comment:
I expected to see a violation raised any time an acispkt is found in a VO backstop but don't see this is done.  Can you confirm?

Reply:
The purpose is not to check if the VO file has ACISPKT commands. The purpose is to not crash searching for ACISPKT commands when there aren't any. Remember that in an assembled history, there can be ECS measurements and/or power commands set via CAP so there MAY actually be ACISPKT commands within the VO time period but which did not appear in the VO*.backstop file.